src/services/youtube.js

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+// import videos from '../dummyData/videos';
+
+export const baseURL = 'https://www.googleapis.com/youtube/v3';
+
+const service = () => {
+  const apiKey = 'AIzaSyCxpoupiS2E_fiFWXR5mfOTG0JNVS_4T1A';
```

Comment:
For security reasons it would've been better if you have stored your apiKey on an environment variable: https://create-react-app.dev/docs/adding-custom-environment-variables/